csi-driver/install.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
```

Comment:
Are the files in `csi-driver/deploy` cloned from some repo or written by you?

Reply:
Just curious

Reply:
All the files in deploy is grabbed from https://github.com/kubernetes-csi/csi-driver-host-path/tree/master/deploy